ServiceDocumentSampleComponents/force-app/main/default/triggers/ServiceDocumentCreation.trigger

```diff
@@ -0,0 +1,143 @@
+trigger ServiceDocumentCreation on DocumentRecipient (after insert) {
+
+    //TODO: 
+    //1. if you have a template with more than 4 signatures, please add it here
+    //2. please edit the content of the sets bellow to contain your template ids accordingly
+    Set<String> WO_TEMPLATES_WITH_SIGNATURES_1 = new Set<String>{''};
+    Set<String> WO_TEMPLATES_WITH_SIGNATURES_2 = new Set<String>{''};
+    Set<String> WO_TEMPLATES_WITH_SIGNATURES_3 = new Set<String>{''};
+    Set<String> WO_TEMPLATES_WITH_SIGNATURES_4 = new Set<String>{''};
+
+    Set<String> WOLI_TEMPLATES_WITH_SIGNATURES_1 = new Set<String>{''};
+    Set<String> WOLI_TEMPLATES_WITH_SIGNATURES_2 = new Set<String>{''};
+    Set<String> WOLI_TEMPLATES_WITH_SIGNATURES_3 = new Set<String>{''};
+    Set<String> WOLI_TEMPLATES_WITH_SIGNATURES_4 = new Set<String>{''};
+
+    Set<String> SA_TEMPLATES_WITH_SIGNATURES_1 = new Set<String>{''};
+    Set<String> SA_TEMPLATES_WITH_SIGNATURES_2 = new Set<String>{''};
+    Set<String> SA_TEMPLATES_WITH_SIGNATURES_3 = new Set<String>{'0M0xx0000004CFUCA2'};
+    Set<String> SA_TEMPLATES_WITH_SIGNATURES_4 = new Set<String>{''};
+
+    final Integer DEFAULT_NUM_SIGNATURES = 3;
```

Comment:
Add a comment for what this variable does, because it is another thing the customer can configure

// TODO: Set threshold for when you want to fire the trigger for when a template not added to an above set
// Eg: If you set this to 3, then if we cannot match the template being used, we'll automatically fire the createServiceDocument invocable action when 3 signatures are collected